lib/main.dart

```diff
@@ -179,7 +179,7 @@ class KAGAppState extends State<KAGApp> with TickerProviderStateMixin {
   }
 
   static AppTypeState getDefaultAppType() {
-    AppType type = AppType.LOGGED_OUT;
+    AppType type = AppType.MOBILE_SITE; // TODO muss vor mergen geandert werden!
```

Comment:
.

Und der bessere Weg zu testen wäre (glaube ich) per m.localhost auf die Seite zuzugreifen. Falls das nicht klappt vorher in der /etc/hosts anlegen